src/js/store/Store.js

```diff
@@ -0,0 +1,41 @@
+import reactiveState from '../core/reactive/reactiveState.js';
+
+export default class Store {
+  constructor({ state, getter, mutation, action }) {
+    this.state = reactiveState(state);
+    this.getters = getter;
+    this.actions = action;
+    this.mutations = mutation;
+
+    this.getter = this.getter.bind(this);
+    this.dispatch = this.dispatch.bind(this);
+    this.commit = this.commit.bind(this);
+  }
```

Comment:
멤버변수는
```suggestion
  state, getters, actions, mutations;
  
  constructor({ state, getter, mutation, action }) {
    this.state = reactiveState(state);
    this.getters = getter;
    this.actions = action;
    this.mutations = mutation;
    
    // bind가 꼭 필요할까요?
    this.getter = this.getter.bind(this);
    this.dispatch = this.dispatch.bind(this);
    this.commit = this.commit.bind(this);
  }
```

멤버변수는 클래스 필드에 미리 정의해주세요!
그래야 이 코드를 보는 다른 사람이 어떤 멤버변수가 사용되는지 파악하기가 쉽답니다 ㅎㅎ